code/modules/mob/living/carbon/carbon_defines.dm

```diff
@@ -58,6 +58,8 @@
 
 	///This is used to determine if the mob failed a breath. If they did fail a brath, they will attempt to breathe each tick, otherwise just once per 4 ticks.
 	var/failed_last_breath = FALSE
+	///Empty breath - Used in the event that [carbon/proc/check_breath()] gets a null breath.
+	var/datum/gas_mixture/empty_breath = new
```

Comment:
this seems wrong

Reply:
Fixed by commit 1071f24ae577340af5936d98c05d9053659182df